DSpace/DSpace
# DS-3734 Missing trigger of item last modified date when adding bitstreams

https://jira.duraspace.org/browse/DS-3734

When a bitstream is added to an item after it was archived the last modified date of the owning item is not triggered.

To reproduce the error:
1. Go to http://demo.dspace.org/xmlui/
2. Log in as administrator
3. Click on any item and hit "Edit this Item"
4. Click on "Item Bitestreams"
5. Click on "Upload a new Bitstream"
6. Upload the file
7. Click on Item Status

Expected behavior: Last Modified equals current date and time. 
Observed behavior: Last Modified didn't change

I should mention that this pull request is made on a branch from dspace-6_x. I just changed `org.dspace.content.BundleServiceImpl#addBitstream(Context context, Bundle bundle, Bitstream bitstream)`.
